serving/docker/trtllm-container-requirements.txt

```diff
@@ -0,0 +1,2 @@
+tensorrt_llm==0.17.0.post1
```

Comment:
remove `tensorrt_llm`
1. this is duplicate with the one in dockerfile
2. this requires `--extra-index-url https://pypi.nvidia.com`